.github/actions/setup-terraform/action.yaml

```diff
@@ -10,6 +10,18 @@ inputs:
   batch-container-image-name:
     description: The name of the container image to use for the Batch job.
     required: true
+  batch-job-definition-vcpu:
+    description: >
+      Count of cVPUs to provision for the container. Per AWS requirements,
+      must be formatted as a float, e.g. 1.0 for 1 vCPU. The minimum
+      is 1 vCPU and values must be specified in increments of 0.25.
+    required: true
+  batch-job-definition-memory:
+    description: Count of megabytes of RAM to provision for the container.
+    required: true
+  repo:
+    description: The name of the repo, to use for namespacing Terraform configs.
+    required: true
```

Comment:
**question:** Can this not be derived from the GitHub context variables within the action itself? Why make it a variable?

Reply:
Oops, you're right! I assumed the composite action wouldn't have access to the `github` environment context but I see now that it does 👍🏻